src/main/java/co/omise/Serializer.java

```diff
@@ -53,19 +51,13 @@ public static Serializer defaultSerializer() {
     private final DateTimeFormatter localDateFormatter;
 
     private Serializer() {
-        dateTimeFormatter = ISODateTimeFormat.dateTimeNoMillis();
-        localDateFormatter = ISODateTimeFormat.date();
+        dateTimeFormatter = DateTimeFormatter.ISO_OFFSET_DATE_TIME;
+        localDateFormatter = DateTimeFormatter.ISO_DATE;
```

Comment:
What's the difference between these two?

Reply:
the `dateTimeFormatter` use for serialize/deserialize date with timezone e.g. `2018-01-01T12:00:00Z` but `localDateFormatter` use for serialize/deserialize only date e.g. `2018-01-01`.

Reply:
👍

Reply:
I think the `Instant` class may be more suitable for storing Date Time here in our SDK